### PR TITLE
Add iOS CLB/MESH AVPlayer plugin samples

### PR DESCRIPTION
**Pull request by borisborgobello**
We now possess an AVPlayer plugin for CDN Load Balancer and Mesh delivery.
This PR :
- Cloned AVPlayer sample for both orch/mesh into a new AVPlayerPlugin folder
- Updated the Podfiles of both repo so that they import LumenMeshAVPlayerPlugin & LumenCDNLoadBalancerAVPlayerPlugin
- Updated AppDelegate to use LMDeliveryClientPlugin.initializeApp() instead
- Updated player view controller to use the Plugin instead of the SDK
- Removed the interactors as they are now embedded inside the plugin

### CODE REVIEW

**Review by tri170391 — APPROVED** [review]

File: mesh/ios/AVPlayerPlugin/AVPlayerMesh/AppDelegate.swift [hunk 1]
```diff
@@ -1,13 +1,22 @@
 //
 //  AppDelegate.swift
 //  AVPlayerMesh
+//
+//  Created by Mehala, Damien on 14/01/2022.
+//
```
Comment (nqkhang) [inline mesh/ios/AVPlayerPlugin/AVPlayerMesh/AppDelegate.swift:6]: I think we wanted to removes these kind of lines?

File: mesh/ios/AVPlayerPlugin/AVPlayerMeshTV/AppDelegate.swift [hunk 2]
```diff
@@ -25,4 +36,7 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
   func applicationDidBecomeActive(_ application: UIApplication) {
     // Restart any tasks that were paused (or not yet started) while the application was inactive. If the application was previously in the background, optionally refresh the user interface.
   }
+
+
```
Comment (nqkhang) [inline mesh/ios/AVPlayerPlugin/AVPlayerMeshTV/AppDelegate.swift:40]: empty lines, should be removed? 👀

File: orchestrator/Apple/AVPlayerPlugin/AVPlayerOrchestrator/AppDelegate.swift [hunk 3]
```diff
@@ -0,0 +1,20 @@
+//
+//  AppDelegate.swift
+//  AVPlayerOrchestrator
+//
+//  Created by Ndiaye, Lamine on 19/08/2020.
```
Comment (nqkhang) [inline orchestrator/Apple/AVPlayerPlugin/AVPlayerOrchestrator/AppDelegate.swift:5]: update this as well

File: mesh/ios/AVPlayerPlugin/AVPlayerMesh/PlayerViewController.swift [hunk 4]
```diff
@@ -1,32 +1,21 @@
 //
 //  PlayerViewController.swift
 //  AVPlayerMesh
+//
+//  Created by Mehala, Damien on 14/01/2022.
```
Comment (nqkhang) [inline mesh/ios/AVPlayerPlugin/AVPlayerMesh/PlayerViewController.swift:5]: ditto (update this line)

File: mesh/ios/AVPlayerPlugin/AVPlayerMesh/ViewController.swift [hunk 5]
```diff
@@ -1,11 +1,14 @@
 //
 //  ViewController.swift
 //  AVPlayerMesh
+//
+//  Created by Mehala, Damien on 14/01/2022.
```
Comment (nqkhang) [inline mesh/ios/AVPlayerPlugin/AVPlayerMesh/ViewController.swift:5]: ditto

File: mesh/ios/AVPlayerPlugin/AVPlayerMesh/SceneDelegate.swift [hunk 6]
```diff
@@ -1,6 +1,9 @@
 //
 //  SceneDelegate.swift
 //  AVPlayerMesh
+//
+//  Created by Mehala, Damien on 14/01/2022.
```
Comment (nqkhang) [inline mesh/ios/AVPlayerPlugin/AVPlayerMesh/SceneDelegate.swift:5]: ditto

File: orchestrator/Apple/AVPlayerPlugin/AVPlayerOrchestrator/SwiftSampleViewController.swift [hunk 7]
```diff
@@ -0,0 +1,67 @@
+//
+//  ViewController.swift
+//  AVPlayerOrchestrator
+//
+//  Created by Ndiaye, Lamine on 19/08/2020.
```
Comment (nqkhang) [inline orchestrator/Apple/AVPlayerPlugin/AVPlayerOrchestrator/SwiftSampleViewController.swift:5]: ditto

**Comment by nqkhang** [comment]
I put some reviews, mainly the `//created by` parts.

Quick confirmation: this does not have the Airplay fix right?

File: mesh/ios/AVPlayerPlugin/AVPlayerMesh/AppDelegate.swift (hunk 1 above)
Comment (borisborgobello) [reply mesh/ios/AVPlayerPlugin/AVPlayerMesh/AppDelegate.swift]: done

File: mesh/ios/AVPlayerPlugin/AVPlayerMeshTV/AppDelegate.swift (hunk 2 above)
Comment (borisborgobello) [reply mesh/ios/AVPlayerPlugin/AVPlayerMeshTV/AppDelegate.swift]: done

**Comment by borisborgobello** [comment]
> I put some reviews, mainly the `//created by` parts.
> 
> Quick confirmation: this does not have the Airplay fix right?

The AirplayManager is available, but indeed we're not using it so it just has the simple Lamine fix. For the AirplayManager we might consider creating another ViewController to show an example usage later.

**Pull request merged**